views/handlers/IslandoraBatchViewsFieldHandlerAction.inc

```diff
@@ -248,6 +248,14 @@ class IslandoraBatchViewsFieldHandlerSetActions extends IslandoraBatchViewsField
         )),
         'query' => drupal_get_destination(),
       ),
+      array(
+        'title' => t('Delete ingested objects'),
```

Comment:
Somewhat about the nomenclature: Should we instead be referring to this action as a "Purge", since to "Delete" may also refer to setting the (object) state to `Deleted` (from `Active`/`Inactive`)... Dunno... Will people get this mixed up? I feel like we probably use "delete" in some other places... Shrug?